source/components/molecules/FilePicker/imageUpload.ts

```diff
@@ -1,15 +1,64 @@
-import { Alert } from "react-native";
-import type { ImageOrVideo } from "react-native-image-crop-picker";
+import { Alert, Linking, Platform } from "react-native";
 import ImagePicker from "react-native-image-crop-picker";
 import uuid from "react-native-uuid";
 
-import type { Image } from "../ImageDisplay/ImageDisplay";
+import type {
+  ImageOrVideo,
+  PickerErrorCode,
+} from "react-native-image-crop-picker";
+import { splitFilePath } from "../../../helpers/FileUpload";
 
 import type { AllowedFileTypes } from "../../../helpers/FileUpload";
-import { splitFilePath } from "../../../helpers/FileUpload";
+import type { Image } from "../ImageDisplay/ImageDisplay";
+
+interface ImageUploadError {
+  code: PickerErrorCode;
+}
+
+const NO_IMAGE_UPLOAD_PERMISSION_CODES = [
+  "E_NO_CAMERA_PERMISSION",
+  "E_NO_LIBRARY_PERMISSION",
+];
+
+const SKIP_WARNING_CODES = ["E_PICKER_CANCELLED"];
 
 const MAX_IMAGE_SIZE_BYTES = 7 * 1000 * 1000;
 
+function showToBigFileAlert() {
+  Alert.alert(
+    "Ogiltiga bilder",
+    "Några av de angivna bilder är för stora. Varje bild får max vara 7 Mb."
+  );
+}
+
+async function goToDeviceSettings() {
+  if (Platform.OS === "android") {
+    await Linking.openSettings();
+  }
+
+  if (Platform.OS === "ios") {
+    await Linking.openURL("app-settings:MittHelsingborg");
```

Comment:
Does `Linking.openSettings();` not work for iOS? I can't find anything in the docs. The worry is that `MittHelsingborg` can be slightly changed and it would break this.

Reply:
Worked on both ios and android. Nice catch! Fixed